Sources/xcodeproj/Objects/Project/PBXObjectReference.swift

```diff
@@ -12,6 +12,9 @@ class PBXObjectReference: NSObject, Comparable, NSCopying {
     /// Weak reference to the objects instance that contains the project objects.
     weak var objects: PBXObjects?
 
+    /// A weak reference to the object
+    weak var objectRef: PBXObject?
```

Comment:
Nitpick: What about calling this object?

Reply:
Yeah I wanted to but it clashed with `func object()`

Reply:
I'd keep this one as object then and rename the other. What do you think?

Reply:
Yeah, was thinking rename `object()` to `getObject()`